srml/session/src/lib.rs

```diff
@@ -265,6 +267,16 @@ decl_storage! {
 	}
 	add_extra_genesis {
 		config(keys): Vec<(T::AccountId, T::Keys)>;
+		build(|
+			  storage: &mut primitives::StorageOverlay,
+			  _: &mut primitives::ChildrenStorageOverlay,
+			  _: &GenesisConfig<T>
+		| {
+			with_storage(storage, || {
+				assert_eq!(<Validators<T>>::get(), vec![]);
```

Comment:
why would you ever want to do this?

Reply:
@bkchr wanted to check that the validators/keys are not set by the staking module before the session module to avoid overwriting the values. Can I please just have one reviewer? xD